src/dysh/spectra/spectrum.py

```diff
@@ -818,6 +929,14 @@ def fake_spectrum(cls, nchan=1024, **kwargs):
         nchan : int, optional
             Number of channels. The default is 1024.
 
+        seed : {None, int, array_like[ints], SeedSequence, BitGenerator, Generator}, optional
```

Comment:
These should be e.g., `~numpy.random.SeedSequence`  so the docs will link to the numpy docs/